packages/astro/CHANGELOG.md

```diff
@@ -415,18 +447,10 @@ The **Astro v1.0.0 Release Candidate** comes includes new features, tons of bug
 
   Astro supports streaming in its templates. Any time Astro encounters an async boundary it will stream out HTML that occurs before it. For example:
 
-  ```astro
-
-  ```
-
```

Comment:
Erm... prettier removed the content here so I just removed the block entirely? Can probably dig up the old content, though.

Reply:
ack, that's odd. cc @Princesseuh to take a look in case this is happening to users.

Reply:
Taking a look, but I honestly have no idea what happened. The plugin ran fine for multiple formats until https://github.com/withastro/astro/pull/4159/files#diff-e3d0a49bea0efda551d3bd7e5e860ab65409efbb9a419d274b5806ccacdd4061 where it broke unexplainably 🤔